source/_integrations/group.markdown

```diff
@@ -4,6 +4,7 @@ description: Instructions on how to setup groups within Home Assistant.
 ha_category:
   - Organization
 ha_release: pre 0.7
+ha_iot_class: Local Push
```

Comment:
A group's state is determined by its members, so I think calculated would be more fitting here.

```suggestion
ha_iot_class: Calculated
```